beanstalk_worker/worker_start_build.py

```diff
@@ -54,36 +54,38 @@ def notify_build_failure(namespace, notify_email, logs):
 def start_build(job_details):
     try:
         debug_log(" Retrieving namespace")
-        name = job_details['name']
-        tag = job_details['tag']
+        appid = job_details['appid']
+        jobid = job_details['jobid']
+        desired_tag = job_details['desired_tag']
+        namespace = appid+"-"+jobid+"-"+desired_tag
```

Comment:
spaces around operators?

we need to ensure pep8 guidelines along with PRs to minimize the effort of fixing such coding standards throughout code at once, if we will keep on taking care of them along side, the code will eventually get clean.